tutoraspects/templates/aspects/jobs/init/clickhouse/init-clickhouse.sh

```diff
@@ -51,7 +51,7 @@ GRANT {{ ON_CLUSTER }} SELECT ON {{ ASPECTS_EVENT_SINK_DATABASE }}.* TO '{{ ASPE
 GRANT {{ ON_CLUSTER }} CREATE TABLE, DROP TABLE, CREATE VIEW, DROP VIEW, SELECT, INSERT, UPDATE, DELETE ON {{ DBT_PROFILE_TARGET_DATABASE }}.* TO '{{ ASPECTS_CLICKHOUSE_REPORT_USER }}';
 
 GRANT {{ ON_CLUSTER }} SELECT, DROP TABLE, DROP VIEW ON {{ ASPECTS_XAPI_DATABASE }}.* TO '{{ ASPECTS_CLICKHOUSE_REPORT_USER }}';
-GRANT {{ ON_CLUSTER }} INSERT, SELECT ON {{ ASPECTS_EVENT_SINK_DATABASE }}.* TO '{{ ASPECTS_CLICKHOUSE_CMS_USER }}';
+GRANT {{ ON_CLUSTER }} INSERT, SELECT, DELETE ON {{ ASPECTS_EVENT_SINK_DATABASE }}.* TO '{{ ASPECTS_CLICKHOUSE_CMS_USER }}';
```

Comment:
@bmtcril I think we would prefer having a separate user for the user retirement in `openedx-event-sink-clickhouse`.

@pomegranited can you create a new user for this?

Reply:
I don't care much either way about a new user for this as it's entirely a bot user with permissions to overwrite this data already. It doesn't seem particularly difficult, we'd just have to override `ch_auth`, but maybe a little confusing.

Reply:
Ok, I'm going to leave this as-is.